examples/tutorials/curation/plot_2_train_a_model.py

```diff
@@ -0,0 +1,125 @@
+"""
+Training a model for automated curation
+=============================
+
+If the pretrained models do not give satisfactory performance on your data, it is easy to train your own classifier using SpikeInterface.
+"""
+import warnings
+warnings.filterwarnings("ignore")
+from pathlib import Path
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+import spikeinterface.core as si
+import spikeinterface.curation as sc
+import spikeinterface.widgets as sw
+
+
+# Note, you can set the number of cores you use using e.g.
+# si.set_global_job_kwargs(n_jobs = 8)
+
+##############################################################################
+# Step 1: Generate and label data
+# -------------------------------
+#
+# For this tutorial, we will use simulated data to create ``recording`` ``sorting`` objects. We'll
+# create two sorting objects: :code:`sorting_1` is coupled to the real recording, so will contain good
+# units; :code:`sorting_2` is uncoupled, so should produce noise. We'll combine the two into one sorting
```

Comment:
I get this and it's very nice idea, but could be quite mysterious / confusing to less experienced users. Can a sentence be added to make this very explicit? i.e. in sorting_1 the  times will match exactly the spike in the recording. For sorting_2 the spike times will not be matched with the recording so will mostly be random noise.